main.js

```diff
@@ -127,7 +128,13 @@ function renderCard(item) {
 
 function renderCatalog(catalog) {
   return createElement('div', { class: 'container' }, [
-    createElement('div', { class: 'row' }, catalog.items.map((item) => renderCard(item)))
+    createElement('div', { class: 'dropdown' }, [
+      createElement('button', { class: 'btn dropdown-toggle', type: 'button', id: 'sort-dropdown' }, ['Sort Menu']),
+      createElement('button', { class: 'btn dropdown-item hidden', id: 'low-to-high' }, ['Low to high']),
```

Comment:
You should conditionally set the `hidden` class based on some application state.